astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -709,9 +709,9 @@ def test_aa_high_precision():
     TARGET_AZ, TARGET_EL = 15.032673509886*u.deg, 50.303110133928*u.deg
     TARGET_DISTANCE = 376252883.247239*u.m
 
-    assert_allclose(moon_aa.az, TARGET_AZ, atol=0.1*u.uas, rtol=0)
-    assert_allclose(moon_aa.alt, TARGET_EL, atol=0.1*u.uas, rtol=0)
-    assert_allclose(moon_aa.distance, TARGET_DISTANCE, atol=0.1*u.mm, rtol=0)
+    assert_allclose(moon_aa.az, TARGET_AZ, atol=0.1*u.uas, rtol=3.7e-10)
```

Comment:
@pllim - could you increase the absolute tolerance by a factor 10 instead? That is more in the spirit of the test.... Sorry for the bother!

Reply:
Factor of 10 is `1 * u.uas`, right? That won't cut it.

```python
>>> moon_aa.az
<Longitude 15.03267352 deg>
>>> TARGET_AZ
<Quantity 15.03267351 deg>
>>> moon_aa.az - TARGET_AZ
<Angle 5.44987699e-09 deg>
>>> (moon_aa.az - TARGET_AZ).to(u.uas)
<Angle 19.61955718 uas>
```

Reply:
I updated the tests to relax `atol` to the smallest values that would make the comparison pass (at least locally).

Reply:
OK, that sounds good - we definitely want the tests to pass again!

Reply:
They did pass, but I have no idea what this change means. Is the moon not where it is supposed to be?

![DSI_hdapproach](https://user-images.githubusercontent.com/2090236/114785769-27286980-9d4b-11eb-9d44-3433fb2d2846.png)

Reply:
It is just that with the slightly more accurate way erfa deals with polar motion, implied coordinates differ a bit.